firefox-ios/Ecosia/UI/DesignSystem/EcosiaBorders.swift

```diff
@@ -0,0 +1,15 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+// This class contains all of Ecosia official border radies system values referenced on the link below.
+// https://www.figma.com/design/8T2rTBVwynJKSdY6MQo5PQ/%E2%9A%9B%EF%B8%8F--Foundations?node-id=71-588
+// Only values present on the above link should be added here and you should avoid using any border radius values outside these.
+public struct EcosiaBorders {
```

Comment:
Really well structured 👌 here and below.
Is there a specific need to use the `_` ?

Reply:
Totally agree and am happy with the advantages this convention will bring 💪 .
Do you think it's worth documenting anywhere in our confluence?
Or better a SwiftLint custom rule (on top of the error that by default you'll get by the compiler 😅 )?
I remember something we can add like 🤷‍♂️ (to begin to ignite our own more personalised linting):

```
custom_rules:
  numeric_variable_prefix:
    name: "Underscored numeric variable name"
    included: 
      - firefox-ios/Ecosia/UI/DesignSystem
    regex: '\b(static\s+)?(let|var)\s+(?!_)[0-9]+[a-zA-Z0-9_]*'
    message: "UI Ecosia's Design System's variables and constants must be prefixed with `_`, like `_2s`, `_1s`."
    severity: warning
```

Reply:
Interesting idea! I think for now we can keep it simple and experiment, it might be we find `_` bad to use and decide to iterate on it still. Once we have a more defined approach and have variables for more things we can document and add custom rules indeed 🙂